scss/components/hero-banner/_hero-banner.scss

```diff
@@ -136,11 +139,38 @@ $hero-max-width-lg: 1500px;
       transform: translate(0, 0);
       left: 0;
     }
-    .hero-banner__hgroup {
-      @include grid-media($media-sm-max) {
-        width: 80%;
-        left: 10%;
-        text-align: center;
+  }
+
+  .hero-banner__hgroup {
+    @include grid-media($media-sm-max) {
+      width: 80%;
+      left: 10%;
+      text-align: center;
+    }
+  }
+
+}
+
+// For hero video players that are the first content item on the page, we want
+// the text content to flow below.
+.system-main-block {
+  &:first-child {
+    > div:first-child {
+      > .hero-banner.has-video:first-child {
+        @include grid-media($media-sm-max) {
+          display: flex;
```

Comment:
Flex box made this easy so I went with it.

https://caniuse.com/#search=flex

Reply:
I was able to run 'npm install', 'grunt sass', and cleared drupal and browser caches, but can't get the header to move above the video. Tried in chrome, firefox, and safari. and still doesn't work. Maybe the build didn't work properly...

Reply:
I had the same experience as @ksharp-drupal. I did get some npm-install errors, so might also be that the build didn't work.
I *do* see the header move below the video, but not above.